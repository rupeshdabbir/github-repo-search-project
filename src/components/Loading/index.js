// @flow

import React from 'react';
import 'suitcss-utils-flex/lib/flex.css';
import LoadingSpinner from './loading.svg';

export default function Loading() {
  return (
    <div className="u-flex u-flexJustifyCenter u-flexAlignItemsCenter">
      <LoadingSpinner width={34} height={34} />
    </div>
  );
}
