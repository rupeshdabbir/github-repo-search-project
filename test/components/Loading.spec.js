import React from 'react';
import renderer from 'react-test-renderer';
import {
  StyleSheetTestUtils,
} from 'aphrodite';
import Loading from 'components/Loading';

describe('Component: Loading', () => {

  beforeAll(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterAll(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should render', () => {
    const component = renderer.create(
      <Loading />
    );
    expect(component).toMatchSnapshot();
  });

});
