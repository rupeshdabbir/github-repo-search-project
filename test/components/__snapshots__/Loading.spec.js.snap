// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loading should render 1`] = `
<div
  className="u-flex u-flexJustifyCenter u-flexAlignItemsCenter"
>
  <svg />
</div>
`;
